dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -462,19 +462,43 @@ case class BinaryOutcomeDLCWithSelf(
   def constructClosingTx(
       privKey: ECPrivateKey,
       spendingInfo: BitcoinUTXOSpendingInfo,
-      isLocal: Boolean): Future[Transaction] = {
-    // Construct Closing Transaction
-    val txBuilder = BitcoinTxBuilder(
-      destinations = Vector(
-        TransactionOutput(spendingInfo.output.value,
-                          P2WPKHWitnessSPKV0(privKey.publicKey))),
-      utxos = Vector(spendingInfo),
-      feeRate = feeRate,
-      changeSPK = emptyChangeSPK,
-      network = network
-    )
+      isLocal: Boolean,
+      isWin: Boolean,
+      isToLocal: Boolean): Future[Transaction] = {
```

Comment:
we now have a `isLocal` and `isToLocal`? :thinking:

Reply:
For posterity, `isLocal` is true when the method is called from the "local" view of this DLCWithSelf, while `isToLocal` refers to whether or not this spends the `ToLocalOutput` or not.

Of course, this will be much easier to follow once I separate into two views so that only the local view exists.